inc/composer/class-command.php

```diff
@@ -332,6 +338,63 @@ protected function shell( InputInterface $input, OutputInterface $output ) {
 		return $return_val;
 	}
 
+	protected function db( InputInterface $input, OutputInterface $output ) {
+		$db = $input->getArgument( 'options' )[0] ?? '';
+		$columns = exec( 'tput cols' );
+		$lines = exec( 'tput lines' );
+
+		$base_command_prefix = sprintf(
+			'cd %s; VOLUME=%s COMPOSE_PROJECT_NAME=%s',
+			'vendor/altis/local-server/docker',
+			escapeshellarg( getcwd() ),
+			$this->get_project_subdomain()
+		);
+
+		$base_command = sprintf(
+			"$base_command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d db",
+			$columns,
+			$lines
+		);
+
+		switch ( $db ) {
+			case 'info':
+				// Env variables
+				$env_variables = explode( PHP_EOL, shell_exec( "$base_command printenv" ) );
+				$values = array_reduce( $env_variables, function( $values, $env_variable_text ) {
+					$env_variable = explode( '=', $env_variable_text );
+					$values[ $env_variable[0] ] = $env_variable[1] ?? '';
+					return $values;
+				}, [] );
+
+				// Ports
+
+				// Find the container ID with Docker Compose
+				$db_container_id = shell_exec( "$base_command_prefix docker-compose ps -q db" );
+
+				// Retrrieve the forwarded ports using Docker and the container ID
+				$ports = shell_exec( sprintf( "$base_command_prefix docker ps --format '{{.Ports}}' --filter id=%s", $db_container_id ) );
+				preg_match( '/.*,\s([\d.]+:[\d]+)->.*/', $ports, $ports_matches );
+
+				$db_info = <<<EOT
+<info>Root password</info>:  ${values['MYSQL_ROOT_PASSWORD']}
+
+<info>Database</info>:       ${values['MYSQL_PASSWORD']}
+<info>User</info>:           ${values['MYSQL_USER']}
+<info>Password</info>:       ${values['MYSQL_PASSWORD']}
+
+<comment>Version</comment>:        ${values['MYSQL_VERSION']}
+<comment>Ports</comment>:          ${ports_matches[1]}
+EOT;
+				echo $output->write( $db_info ) . PHP_EOL;
```

Comment:
`echo` isn't needed here

Reply:
Actually why did you need to do the `PHP_EOL` outside of the call to `$output->write()`?

Reply:
It's an old code that I added during development. I just stripped it out